### DISCUSSION TITLE
Are new social icons too big?

### DISCUSSION THREAD

**Discussion by ramiro** [gohugo-ananke/ananke#776] [3 - Ideas]
The new feature is great. The size/style consistency among icons is welcome. I had tried to add a X (ex Twitter) SVG icon myself and never managed to get it to look good.

I am not a designer but to my untrained eyes the size and arrangement (spacing, separation) of the previous ones looked better.

Samples from the example site:

### Before

<img width="507" alt="follow-before" src="https://github.com/user-attachments/assets/d901d051-497c-4fd2-8f58-17073679015c">

### Now

<img width="462" alt="follow-after" src="https://github.com/user-attachments/assets/118d8f55-3052-475b-abe7-e6741cb14037">

### Before

<img width="260" alt="share-before" src="https://github.com/user-attachments/assets/5737e3c9-8fa6-40c7-9813-00660c1545d1">

### Now

<img width="265" alt="share-after" src="https://github.com/user-attachments/assets/99dbd017-4340-41b0-95ea-d9142d28f1bc">

**Comment by davidsneighbour** [comment]
Yep, they are awfully large. 

```css
.ananke-socials svg {
height: 1rem;
width: auto;
}
```

Might solve this (change the 1rem until you like what you see). The spacing is also missing.
